modules/helpers/dice-helpers.js

```diff
@@ -251,3 +251,60 @@ export default class DiceHelpers {
     return dicePool;
   }
 }
+
+/**
+ * Helper function to build a dice pool
+ * @param actor_id ID of the actor making the check
+ * @param skill_name name of the string of the skill
+ * @param incoming_roll existing dice, e.g. difficulty dice
+ * @returns {DicePoolFFG}
+ */
+export function get_dice_pool(actor_id, skill_name, incoming_roll) {
+  let actor = game.actors.get(actor_id);
+  var parsed_skill_name = convert_skill_name(skill_name);
+  var skill = actor.system.skills[parsed_skill_name];
+  var characteristic = actor.system.characteristics[skill.characteristic];
+
+  let dicePool = new DicePoolFFG({
```

Comment:
```suggestion
  const actor = game.actors.get(actor_id);
  const parsed_skill_name = convert_skill_name(skill_name);
  const skill = actor.system.skills[parsed_skill_name];
  const characteristic = actor.system.characteristics[skill.characteristic];

  const dicePool = new DicePoolFFG({
```

Prefer specific immutability wherever possible, as you don't expect these objects to change type after instantiation. Happy to chat about why this is good practice.

Reply:
nope, makes sense